sdk/communication/azure-communication-callautomation/azure/communication/callautomation/__init__.py

```diff
@@ -119,7 +121,7 @@ def __getattr__(name):
         from ._models import ServerCallLocator
         return ServerCallLocator
     if name == 'MicrosoftBotIdentifier':
-        warnings.warn(f"{name} is deprecated and should not be used.", DeprecationWarning)
-        from ._shared.models  import _MicrosoftBotIdentifier
-        return _MicrosoftBotIdentifier
+        warnings.warn(f"{name} is deprecated and should not be used. Please use MicrosoftTeamsAppIdentifier instead.",
+                       DeprecationWarning)
+        return MicrosoftTeamsAppIdentifier
```

Comment:
We can't return a different object here - as it does not have the same constructor, and so can result in breaking changes. We would need to be careful about this.

Reply:
Reverted to returning MSBotIdentifier, with just Deprecation message.